### ISSUE TITLE
Stable release was not created (registry_rebuild)

### ISSUE THREAD

**Issue by webflo** [drupal-composer/drupal-packagist#3]
_From @derhasi on January 21, 2015 8:53_

Currently only 1.\* releases are available for registry_rebuild, althoguh there is an up-to-date recommended 2.2 stable release.
- http://drupal-packagist.webflo.io/packages/drupal/registry_rebuild
- https://www.drupal.org/project/registry_rebuild

_Copied from original issue: webflo/packagist#12_

**Comment by webflo** [comment]
The import process returns the following error messages.

```

Skipped parsing 7.x-2.x, Undefined index: name
Reading composer.json of http://git.drupal.org/project/registry_rebuild (7.1.0)
Importing tag 7.1.0 (7.1.0.0)
Reading composer.json of http://git.drupal.org/project/registry_rebuild (7.1.1)
Importing tag 7.1.1 (7.1.1.0)
Reading composer.json of http://git.drupal.org/project/registry_rebuild (7.1.10)
<warning>Skipped tag 7.1.10, Undefined index: name</warning>
Reading composer.json of http://git.drupal.org/project/registry_rebuild (7.1.11)
<warning>Skipped tag 7.1.11, Undefined index: name</warning>
Reading composer.json of http://git.drupal.org/project/registry_rebuild (7.1.2)
Importing tag 7.1.2 (7.1.2.0)
Reading composer.json of http://git.drupal.org/project/registry_rebuild (7.1.3)
Importing tag 7.1.3 (7.1.3.0)
Reading composer.json of http://git.drupal.org/project/registry_rebuild (7.1.4)
Importing tag 7.1.4 (7.1.4.0)
Reading composer.json of http://git.drupal.org/project/registry_rebuild (7.1.5)
<warning>Skipped tag 7.1.5, Undefined index: name</warning>
Reading composer.json of http://git.drupal.org/project/registry_rebuild (7.1.6)
<warning>Skipped tag 7.1.6, Undefined index: name</warning>
Reading composer.json of http://git.drupal.org/project/registry_rebuild (7.1.7)
<warning>Skipped tag 7.1.7, Undefined index: name</warning>
Reading composer.json of http://git.drupal.org/project/registry_rebuild (7.1.8)
<warning>Skipped tag 7.1.8, Undefined index: name</warning>
Reading composer.json of http://git.drupal.org/project/registry_rebuild (7.1.9)
<warning>Skipped tag 7.1.9, Undefined index: name</warning>
Reading composer.json of http://git.drupal.org/project/registry_rebuild (7.2.0)
<warning>Skipped tag 7.2.0, Undefined index: name</warning>
Reading composer.json of http://git.drupal.org/project/registry_rebuild (7.2.1)
<warning>Skipped tag 7.2.1, Undefined index: name</warning>
Reading composer.json of http://git.drupal.org/project/registry_rebuild (7.2.2)
<warning>Skipped tag 7.2.2, Undefined index: name</warning>
Reading composer.json of http://git.drupal.org/project/registry_rebuild (7.x-1.x)
Skipped branch 7.x-1.x, Undefined index: name

Reading composer.json of http://git.drupal.org/project/registry_rebuild (7.x-2.x)
Skipped branch 7.x-2.x, Undefined index: name
```

**Comment by webflo** [comment]
Bug was fixed in #12.
